assets/blocks/single-course-style-editor.scss

```diff
@@ -1,2 +1,3 @@
 @import 'course-outline/course-outline-editor';
-@import 'course-progress-block/course-progress-editor'
+@import 'course-progress-block/course-progress-editor';
+@import '../css/sensei-course-theme/course-settings-editor';
```

Comment:
About this file, since this folder only handles blocks, I think we could import it in another place.
Maybe we could add it as part of `assets/css/course-editor.scss` that I added [here](https://github.com/Automattic/sensei/pull/4599/commits/cf6afb47e6bda49823d98bc97b4813851701767d) (not merged yet). WDYT?

Reply:
Removed that CSS file altogether. I had setup styles for settings originally in anticipation with very custom designs, but now it looks like those were not needed so much. 👍